browser/html/cool.html.m4

```diff
@@ -298,7 +298,8 @@ m4_ifelse(MOBILEAPP,[true],
       window.frameAncestors = '';
       window.socketProxy = false;
       window.tileSize = 256;
-      window.uiDefaults = {};],
+      window.uiDefaults = {};
+      window.useIntegrationTheme = '%USE_INTEGRATION_THEME%';],
```

Comment:
Yes, but we can't use the %FOO% things for the mobile apps, as in their case the cool.html file does not pass through the FileServer.cpp code, but is used as such. Just hardcode whatever is the suitable value of useIntegrationTheme for the mobile apps.